content/docs/command-reference/plots/diff.md

```diff
@@ -6,76 +6,77 @@ plotting them in a single image.
 ## Synopsis
 
 ```usage
-usage: dvc plots diff [-h] [-q | -v] [-t [TEMPLATE]] [-d [DATAFILE]] [-f FILE]
-                     [-s SELECT] [-x X] [-y Y] [--stdout] [--no-csv-header]
-                     [--no-html] [--title TITLE] [--xlab XLAB] [--ylab YLAB]
-                     [revisions [revisions ...]]
+usage: dvc plots diff [-h] [-q | -v] [-t <path>]
+                      [--targets [<path> [<path> ...]]] [-o <path>]
+                      [-x <field>] [-y <field>] [--no-csv-header]
+                      [--show-vega] [--title <text>]
+                      [--x-label <text>] [--y-label <text>]
+                      [revisions [revisions ...]]
 
 positional arguments:
-  revisions             Git commits to plot from
+  revisions             Git commits to plot from/to
 ```
 
 ## Description
 
-This command visualize difference between metrics among experiments in the
-repository history. Requires that Git is being used to version the metrics
-files.
-
-The metrics file needs to be specified through `-d`/`--datafile` option. Also, a
-plot can be customized with
-[plot templates](/doc/command-reference/plots#plot-templates) using the
-`--template` option. To learn more about the file formats and templates please
-see `dvc plots`.
+This command is a way to visualize the difference between metrics among
+experiments in the <abbr>repository</abbr> history. Requires that the target
+metric files are versioned with Git. These should specified with the `--targets`
```

Comment:
> Requires that the target metric files are versioned with Git.

is is true even for the recent DVC? cc @dmpetrov @efiop @pared

Reply:
I've been QAing plots today and while this text seems close to accurate to me, I improved it a little in 3a418f66.

But see also this related question: https://github.com/iterative/dvc.org/pull/1382#pullrequestreview-425865736